cmd/driver-registrar/main.go

```diff
@@ -110,6 +201,15 @@ func main() {
 	}
 	glog.V(2).Infof("CSI driver node ID: %q", csiDriverNodeId)
 
+	// When kubeletRegistrationPath is specified then driver-registrar ONLY acts
+	// as gRPC server which replies to registration requests initiated by kubelet's
+	// pluginswatcher infrastructure. Node labeling is done by kubelet's csi code.
+	if *kubeletRegistrationPath != "" {
+		stopCh := make(chan bool)
```

Comment:
sorry I am nitpicking so much. But now that we have cleared up this part - If you are spawning 1 goroutine and then blocking the "main thread" indefinitely, why are you not just doing the work in `e.grpcServer.Serve(lis)` in the main thread? There is no need for goroutines here.

Reply:
Before doing `e.grpcServer.Serve(lis)` there are a few prep things are done. I think this way it is just looks a bit easier to read, otherwise all these would need to go into main.